src/kinetics-observer.cpp

```diff
@@ -998,18 +1138,29 @@ std::vector<int> KineticsObserver::getListOfContacts() const
       v.push_back(i);
     }
   }
-  return v;
+  return v.size();
```

Comment:
(instead of the whole function)
```suggestion
  Index out;
  for(const auto & c : contacts_)
  {
    if(c.isSet) { out += 1; }
  }
  return out;
```

Reply:
My bad but `out` should be initialized here: `Index out = 0;`